### PR TITLE
Update remote-update.md

### PR DESCRIPTION
**Pull request by CarHern**
this feature is not available on GCC high tenants so Im calling that out in our documentation.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/devices/remote-update.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Using the Microsoft Teams admin center, you can update your Teams devices, such
 Device firmware updates can either be applied automatically or scheduled for a future date and time. Other available device updates aren't applied automatically, but can be applied manually or scheduled for a future date and time.
 
 > [!NOTE]
-> While device firmware updates can be scheduled, if the scheduled date and time falls after the configured maximum 30 or 90 day delay, the firmware update is applied when the maximum delay is reached. The scheduled date and time are ignored.
+> While device firmware updates can be scheduled, if the scheduled date and time falls after the configured maximum 30 or 90 day delay, the firmware update is applied when the maximum delay is reached. The scheduled date and time are ignored. Additionally updating Microsoft Teams devices Remotely is a feature not yet available on US Goverment Cloud tenants (GCC-High).
```
Comment (JohanFreelancer9) [inline Teams/devices/remote-update.md:30]:
```suggestion
> While device firmware updates can be scheduled, if the scheduled date and time falls after the configured maximum 30 or 90 day delay, the firmware update is applied when the maximum delay is reached. The scheduled date and time are ignored. Additionally, updating Microsoft Teams devices remotely is a feature not yet available on US Government Cloud tenants (GCC-High).
```

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dstrome** [comment]
confirming change with product team.

**Comment by dstrome** [comment]
confirmed. merging pr.

**Pull request merged**